conans/client/grapher.py

```diff
@@ -0,0 +1,39 @@
+import os
+from conans.model.ref import ConanFileReference
+
+class ConanGrapher(object):
+    def __init__(self, project_reference, deps_graph):
+        self._deps_graph = deps_graph
+        self._project_reference = project_reference
+
+    def graph(self):
```

Comment:
I would decouple the functionality of creating the representation, in memory, and saving it in a file. For me the ``ConanGrapher.graph()`` would just output a string with the graph. And the ``conan info --graph`` would receive an optional parameter with the graph name (by default graph.dot), in the form ``conan info --graph=mygraph.dot`` (I am thinking of generating different graphs for different configurations, as dependencies can be conditional). Then I would add a method ``ConanGrapher.graph_file(filename)`` that would call ``graph()`` and save a file to disk with the result.